core/scripts/cli/cli.js

```diff
@@ -0,0 +1,11 @@
+const cli = async function(callback) {
+  try {
+    console.log("You have started the UMA CLI!");
+  } catch (e) {
+    console.log("ERROR: " + e);
```

Comment:
Do we need to call callback(e) here in case of error? Or does that break the bash script somehow?

Reply:
Yup, I just copied this from one of our other scripts, but that's definitely a better pattern so the error bubbles up.